pkg/syncer/types.go

```diff
@@ -45,6 +45,9 @@ const (
 	// key for HealthStatus annotation on PVC
 	annVolumeHealth = "volumehealth.storage.kubernetes.io/health"
 
+	// key for PV to backingDiskObjectId mapping annotation on PVC
+	annPVtoBackingDiskObjectId = "cns.vmware.com"
```

Comment:
Somehow my previous comment was deleted. Repost it here.

I don't believe put PV "uuid" in annotation key is the right thing to do. Add @xing-yang to comment on it.

Instead, it might better to set annPVtoVVOL to "cns.vmware.com/pv-to-vvol-mapping". And moving the PV uuid is to annotation value.
So the PVC annotation will be like,
```
cns.vmware.com/pv-to-vvol-mapping=<pv uuid>:<vvol backing object id>.
```
Additionally, pv uuid might not be necessary, as it is available in VolumeHandle in the bound PV of each PVC. I will leave it to you.

Reply:
This looks good. We need PV uuid so user of this annotation does not have to query PV object.

`cns.vmware.com/pv-to-backingdiskobjectid-mapping=<pv uuid>:<backing disk object id>`

Reply:
Change the annotation format.